data_steward/constants/cdr_cleaner/clean_cdr.py

```diff
@@ -7,6 +7,9 @@
 DEID_BASE = 'deid_base'
 DEID_CLEAN = 'deid_clean'
 FITBIT = 'fitbit'
+CONTROLLED_TIER_DEID = 'controlled_deid'
```

Comment:
I think we should change these strings to `controlled_tier_*` as well. Any thoughts?

Reply:
Oops, forgot to change those. I changed them to `controlled_tier_*` 👍🏾